entity-types/ext-temporal/definition.yml

```diff
@@ -0,0 +1,21 @@
+---
+domain: EXT
+type: TEMPORAL
+synthesis:
+  rules:
+    - identifier: temporal_identifier
+      name: temporal_hostname
+      encodeIdentifierInGUID: true
+      conditions:
+        - attribute: eventType
+          value: Metric
```

Comment:
You provided test data for `CustomEvent` and `Log`, there isn't test data for `Metric`.
As explained in [the documentation](https://github.com/newrelic/entity-definitions/tree/main?tab=readme-ov-file#how-to-add-testing-data): `The file name is the event name reported to New Relic`
So, in this case, the file name for the test data should be `Metric.json`